### DISCUSSION TITLE
Compose Multiplatfor Support?

### DISCUSSION THREAD

**Discussion by PMARZV** [patrykandpatrick/vico#505] [Ideas]
This is the best charts library fro compose by far. Are you planning on adding Compose Multiplatform support? It would be a huge gamechanger! Thanx!

**Comment by yuriykulikov** [comment]
Would also really appreciate this! I am building an app with Compose which runs on a desktop and on Android it it be really cool to have the same charts in both.

**Comment by yuriykulikov** [comment]
https://github.com/JetBrains/compose-multiplatform

**Comment by Gowsky** [comment]
Hello @Gerizard and @yuriykulikov, this topic has been already discussed in #353.
